packages/react-dev-utils/README.md

```diff
@@ -170,7 +170,7 @@ module: {
 
 Captures JS and CSS asset sizes inside the passed `buildFolder`. Save the result value to compare it after the build.
 
-##### `printFileSizesAfterBuild(webpackStats: WebpackStats, previousFileSizes: OpaqueFileSizes)`
+##### `printFileSizesAfterBuild(webpackStats: WebpackStats, previousFileSizes: OpaqueFileSizes, buildFolder: string)`
```

Comment:
Hi! Can you also add `maxBundleGzipSize` and `maxChunkGzipSize` as optional fields?

`maxBundleGzipSize?: number, maxChunkGzipSize?: number`